recipes/usrsctp/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, CMake, tools, AutoToolsBuildEnvironment
+from shutil import copyfile
+import os
+
+
+class ConanRecipe(ConanFile):
+    name = "usrsctp"
+    description = "SCTP user-land implementation (usrsctp)"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sctplab/usrsctp"
+    license = "BSD 3"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

https://spdx.org/licenses/BSD-3-Clause.html